src/Util/EMailer/MailBuilder.php

```diff
@@ -42,11 +45,12 @@ abstract class MailBuilder
 	/** @var int */
 	protected $recipientUid = null;
 
-	public function __construct(L10n $l10n, BaseURL $baseUrl, IConfig $config)
+	public function __construct(L10n $l10n, BaseURL $baseUrl, IConfig $config, LoggerInterface $logger)
 	{
 		$this->l10n    = $l10n;
 		$this->baseUrl = $baseUrl;
 		$this->config  = $config;
+		$this->logger = $logger;
```

Comment:
You've aligned `l10n`, now you have to align `logger` as well.

Reply:
Done :)
https://github.com/friendica/friendica/pull/8215/commits/3e898d04157f323a399515b13e22803e436f247b#diff-9e19dcc5d5869063bf258b9089f8ee78R53